tests/lib/rules/padding-around-describe-blocks.spec.js

```diff
@@ -0,0 +1,150 @@
+/**
```

Comment:
It turns out these tests didn't need to change to check the around functionality. hooray.

Reply:
Nope, that can't be true. I think i need to add a statement after a describe. one sec.

Reply:
Yes, to test the after I needed something other than a `describe` following a `describe`.